### DISCUSSION TITLE
Too many arguments

### DISCUSSION THREAD

**Discussion by theking2** [community/community#41699] [GitHub Education]
Students get a Too many arguments error when opening assignments in vscode. What is the remedy?

**Comment by samus-aran** [comment]
Hi @theking2 👋 

We're happy you're here.

I'm sure other community members are more savvy than myself at troubleshooting code. I have found an [older discussion (thread)](https://education.github.community/t/failed-to-run-git-clone-fatal-too-many-arguments-with-vs-code-extension/72478) that someone has a similar error which they almost narrowed it down that the student repo names were not being escaped. Do you see any similarities?

Are all your students receiving this error? Do you also happen to have screenshots to share that could assist the team?

**Reply by theking2** [reply]
Older discussions dont help here. They are no longer monitored/active. Not my decision but theirs.

**Comment by theking2** [comment]
Not all. It probably is due to the fact the Documents folder is used and Documents might sit on a OneDrive base folder. Please _test this_ in a real world situation and come back to me with possible solutions.

----------------------------------------

14 dec. 2022 23:21:46 Ashley Nicolson ***@***.***>:
